src/main/java/racing/Game.java

```diff
@@ -0,0 +1,54 @@
+package racing;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Game {
+    private GameInput input = new GameInput();
+
+    private GameOutput output = new GameOutput();
+
+    public void play() {
+        output.printAskCarCount();
+        final int carCount = input.carCount();
+        output.printCarCount(carCount);
+
+        List<Car> cars = this.carSetting(carCount);
+
+        output.printAskRoundCount();
+        final int roundCount = input.roundCount();
+        this.playRace(roundCount, cars);
+
+    }
```

Comment:
```suggestion
        this.playRace(roundCount, cars);
    }
```
개행도 의미 있게 사용하는 것이 좋습니다. 문맥에 따라서 개행을 나눌 수는 있지만 의미 없는 개행은 자제하는 것이 좋아요.